product/02-innersource-and-agile.md

```diff
@@ -0,0 +1,123 @@
+# InnerSource and Agile
+
+You want to improve your product and deliver those improvements faster to customers.
+You want to make stakeholders happy.
+InnerSource helps your team deliver value and maintain autonomy especially in a highly interconnected world.
+
+
+## Autonomous teams in an interconnected world
+
+Organisations try to deliver value to customers quickly.
+One common cause for delays are dependencies in the delivery process.
+As a result many organisations have moved to cross functional teams where communication with customers, design, implementation, testing and deployment/ operations are all in one team thus eliminating the need for costly handovers.
+However in order to achieve high performance teams try to also eliminate waste and re-use existing components.
+From a team perspective each re-used component however adds another link, another dependency outside of the control of that team.
+Often the result are lengthy roadmap discussions to get everyone on the same page and globally optimise prioritization so that changes that are needed in central components can be implemented in an orderly fashion.
+What's more often there are so many requests coming from teams that one central component team runs out of capacity to implement all of the requested changes.
+
+While in traditional organizations there are only two ways of making changes to dependencies:
+* Submit a feature request/ bug report and wait for the other team to prioritize that change and implement it.
+* Build a workaround to avoid the bug or locally provide the functionality needed.
+If none of those options is successful typically the issue is being escalated and decided at a higher hierarchical level-
```

Comment:
```suggestion

If none of those options is successful typically the issue is being escalated and decided at a higher hierarchical level-
```
Extra newline prevents this sentence from being included in the preceding bullet.